src/wallet/walletdb.cpp

```
@@ -1186,9 +1186,11 @@ std::unique_ptr<WalletDatabase> CreateDummyWalletDatabase()
 /** Return object for accessing temporary in-memory database. */
 std::unique_ptr<WalletDatabase> CreateMockWalletDatabase()
 {
-#ifdef USE_BDB
+#ifdef USE_SQLITE
     return std::make_unique<BerkeleyDatabase>(std::make_shared<BerkeleyEnvironment>(), "");
-#elif defined(USE_SQLITE)
+#elif defined(USE_BDB)
     return std::make_unique<SQLiteDatabase>("", "", true);
+#elif
```

Cppcheck (2.10-2)

[error] Syntax error in #elif

(preprocessorErrorDirective)